### PR TITLE
Upgrade to Grails 5

### PR DESCRIPTION
**Pull request by sbearcsiro**

### CODE REVIEW

File: grails-app/services/au/org/ala/images/AnalyticsService.groovy [hunk 1]
```diff
@@ -113,25 +110,29 @@ class AnalyticsService {
...
-                        }
+                    HttpURLConnection connection = (HttpURLConnection) queryURL.toString().toURL().openConnection()
+                    connection.requestMethod = 'POST'
+                    connection.setRequestProperty('Content-Type', 'application/x-www-form-urlencoded')
+                    def formBody = toFormBody(requestBody).getBytes(StandardCharsets.UTF_8)
+                    connection.doOutput = true
+                    connection.outputStream.withStream {
+                        it.write(formBody)
+                    }
+                    def responseCode = connection.responseCode
```
Comment (adam-collins) [inline grails-app/services/au/org/ala/images/AnalyticsService.groovy:122]: Does the outputStream need to be closed?

**Review by adam-collins — COMMENTED** [review]
One comment and OK except for failing tests

File: grails-app/services/au/org/ala/images/AnalyticsService.groovy (hunk 1 above)
Comment (sbearcsiro) [reply grails-app/services/au/org/ala/images/AnalyticsService.groovy:122]: `.withStream` should handle closing the stream

**Comment by yasima-csiro** [comment]
This will be closed since Grails 6 branch will include these changes.

**Pull request closed without merging**